sensors/nxt_i2c_sensor_defs.c

```diff
@@ -513,7 +513,7 @@ const struct nxt_i2c_sensor_info nxt_i2c_sensor_defs[] = {
 		 */
 		.name		= LEGO_POWER_STORAGE_SENSOR_NAME,
 		.vendor_id	= "LEGO",
-		.product_id	= "", /* LMS2012 fakes this with "Store." */
+		.product_id	= "Store",
```

Comment:
"Store" confirmed against [LMS2012](https://github.com/mindboards/ev3sources/blob/master/lms2012/lms2012/Linux_AM1808/sys/settings/typedata.rcf#L87). The period in 'Temp." appears to be a place holder for 5 byte identifier.